test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -150,57 +151,24 @@ public static WebArchive deploy() {
     public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployment(LRA_PARTICIPANT_DEPLOYMENT_QUALIFIER) URL baseURL)
             throws URISyntaxException {
 
-        String lraId;
-        URI lraListenerURI = UriBuilder.fromUri(baseURL.toURI()).path(LRAListener.LRA_LISTENER_PATH).build();
-
-        // Starts an LRA with a short time limit by invoking a resource annotated with @LRA. The time limit is
-        // defined as LRAListener.LRA_SHORT_TIMELIMIT
-        Response response = client
-                .target(lraListenerURI)
-                .path(LRAListener.LRA_LISTENER_ACTION)
-                .request()
-                .put(null);
-
-        Assert.assertEquals("LRA participant action", 200, response.getStatus());
+        Long SHORTER_TIMEOUT = 1000L;
+        URI shortLRA = lraClient.startLRA(null, "Short Timeout Recovery Test", SHORTER_TIMEOUT, ChronoUnit.MILLIS);
```

Comment:
Was the reason to not use the LRAListener.LRA_LISTENER_ACTION to start the transaction something to do with efficiency and just start the LRA directly with the coordinator but with no participant registered or something like that?

Reply:
If you would like to add a new test that is OK with me but on the other hand personally I think it's OK just to have explained the reason for the change in this comment thread on the pull request

Reply:
I am going to resolve this thread as I am not personally asking for a new test to be added (but if you would like to that is fine with me :) )